### PR TITLE
Code quality instrumentation

### PR DESCRIPTION
**Pull request by cschroed-usgs**
Adopting the maven code quality parent pom, enabling its usage in Travis, and enabling its usage in the the containerized Jenkins build.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -35,4 +40,4 @@
           - sudo sh docker-bench-security.sh -c container_images
           - docker run -d --name db arminc/clair-db
           - docker run -p 6060:6060 --link db:postgres -d --name clair arminc/clair-local-scan:v2.0.3
-          - clair-scanner -c "http://127.0.0.1:6060" --ip "$(ip -4 addr show eth0 | grep -oP '(?<=inet\s)\d+(\.\d+){3}')" $DOCKER_IMAGE:latest || true
+          - clair-scanner -c "http://127.0.0.1:6060" --ip "$(ip -4 addr show docker0 | grep -oP '(?<=inet\s)\d+(\.\d+){3}')" $DOCKER_IMAGE:latest || true
```
Comment (cschroed-usgs) [inline .travis.yml]: unrelated bug fix for clair scanner

File: Dockerfile [hunk 2]
```diff
@@ -5,14 +5,31 @@ FROM maven@sha256:b37da91062d450f3c11c619187f0207bbb497fc89d265a46bbc6dc5f17c02a
...
+ARG SONAR_HOST_URL
+ENV SONAR_HOST_URL=$SONAR_HOST_URL
+
+ARG SONAR_LOGIN
+ENV SONAR_LOGIN=$SONAR_LOGIN
+
+RUN if getent ahosts "sslhelp.doi.net" > /dev/null 2>&1; then \
+		wget 'http://sslhelp.doi.net/docs/DOIRootCA2.cer' && \
+		keytool -import -trustcacerts -file DOIRootCA2.cer -alias DOIRootCA2.cer -keystore $JAVA_HOME/jre/lib/security/cacerts -noprompt -storepass changeit; \
+	fi
```
Comment (cschroed-usgs) [inline Dockerfile]: This little scriptlet is not related to code quality; just trying to get the build to pass on-network.

**Comment by zmoore-usgs** [comment]
@cschroed-usgs I've pulled these changes into my PR here which had a bunch of merge conflicts with this one due to my upgrade to Spring Boot 2.x: https://github.com/USGS-CIDA/aqcu-svp-report/pull/21

**Comment by cschroed-usgs** [comment]
Ah sorry for the conflicts. Thanks for resolving them. It looks like everything is present in the other PR, so let's close this one.
